### PR TITLE
Replace warning about viewing materials making you ineligible to participate in DART

### PR DESCRIPTION
**Pull request by rosemm**
I replaced the warnings in each of the "new to..." files. We've changed the exclusion criteria for the DART research program, so folks who have seen our materials will no longer be ineligible to participate. 
I switched it from a warning box to "cool-fact" (icon is a brain), and removed the "don't look at these modules!" language and replaced with a carrot to encourage interested folks to click the links above ("If you meet our research criteria and decide to enroll as a study participant, you'll receive additional data science learning support, including a customized curriculum of materials and the peership and support of a community of learners like you.")

### CODE REVIEW

**Comment by franzenr** [comment]
bullets don't seem to be working on my end. I feel like the whitespace problems frequently go back and forth, and I'm not sure where things land currently, but I'm wondering if that could be it?

**Comment by rosemm** [comment]
Hmm I can't replicate the bullet issue (e.g. https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/rose_fix/new_to_data_science.md renders fine for me). Is there a particular page where you were noticing that? @franzenr

**Comment by franzenr** [comment]
Hmm, I'm having that issue even on that exact link... maybe it's a caching thing for me? I'll see what happens in private browser, etc 
screenshot demonstrating highlight box issue where bullets just appear as asterisks and all appear as if on one continuous line.

File: new_to_data_science.md [hunk 1]
```diff
@@ -16,17 +16,14 @@ First, some various and sundry things that might interest you, beginning with an
 
 This grant includes the creation of dozens of modules that are aimed at being 1 hour or less in duration and each have a narrow focus and clear learning objectives.  They are asynchronous and you can take them at any time!
 
-<div class = "warning">
+<div class = "cool-fact">
```
Comment (rosemm) [inline new_to_data_science.md:19]:
```suggestion
<div class = "cool-fact">

```

File: new_to_data_science.md [hunk 2]
```diff
@@ -16,17 +16,14 @@ First, some various and sundry things that might interest you, beginning with an
...
 * Our interest form at https://redcap.link/dart-interest
 * Taking the screening for participation in the study at https://redcap.link/DART-survey
 * Emailing us at dart@chop.edu
 
-and consider **not** consuming these modules.
-
-Consuming these modules will **render you ineligible to participate** in our research.  However, we know that they're useful on their own, outside of our conduct of research, so we're fast-tracking access to these materials.
-
+If you meet our research criteria and decide to enroll as a study participant, you'll receive additional data science learning support, including a customized curriculum of materials and the peership and support of a community of learners like you. 
 </div>
```
Comment (rosemm) [inline new_to_data_science.md:27]:
```suggestion

</div>
```

File: new_to_python.md [hunk 3]
```diff
@@ -48,17 +48,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
 
 This grant includes the creation of dozens of modules that are aimed at being 1 hour or less in duration and each have a narrow focus and clear learning objectives.  They are asynchronous and you can take them at any time!
 
-<div class = "warning">
+<div class = "cool-fact">
```
Comment (rosemm) [inline new_to_python.md:51]:
```suggestion
<div class = "cool-fact">

```

File: new_to_python.md [hunk 4]
```diff
@@ -48,17 +48,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
...
 * Our interest form at https://redcap.link/dart-interest
 * Taking the screening for participation in the study at https://redcap.link/DART-survey
 * Emailing us at dart@chop.edu
 
-and consider **not** consuming these modules.
-
-Consuming these modules will **render you ineligible to participate** in our research.  However, we know that they're useful on their own, outside of our conduct of research, so we're fast-tracking access to these materials.
-
+If you meet our research criteria and decide to enroll as a study participant, you'll receive additional data science learning support, including a customized curriculum of materials and the peership and support of a community of learners like you. 
 </div>
```
Comment (rosemm) [inline new_to_python.md:59]:
```suggestion

</div>
```

File: new_to_r.md [hunk 5]
```diff
@@ -68,16 +68,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
 
 This grant includes the creation of dozens of modules that are aimed at being 1 hour or less in duration and each have a narrow focus and clear learning objectives.  They are asynchronous and you can take them at any time!
 
-<div class = "warning">
+<div class = "cool-fact">
```
Comment (rosemm) [inline new_to_r.md:71]:
```suggestion
<div class = "cool-fact">

```

File: new_to_sql.md [hunk 6]
```diff
@@ -38,17 +38,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
 
 This grant includes the creation of dozens of modules that are aimed at being 1 hour or less in duration and each have a narrow focus and clear learning objectives.  They are asynchronous and you can take them at any time!
 
-<div class = "warning">
+<div class = "cool-fact">
```
Comment (rosemm) [inline new_to_sql.md:41]:
```suggestion
<div class = "cool-fact">

```

File: new_to_sql.md [hunk 7]
```diff
@@ -38,17 +38,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
...
 * Our interest form at https://redcap.link/dart-interest
 * Taking the screening for participation in the study at https://redcap.link/DART-survey
 * Emailing us at dart@chop.edu
 
-and consider **not** consuming these modules.
-
-Consuming these modules will **render you ineligible to participate** in our research.  However, we know that they're useful on their own, outside of our conduct of research, so we're fast-tracking access to these materials.
-
+If you meet our research criteria and decide to enroll as a study participant, you'll receive additional data science learning support, including a customized curriculum of materials and the peership and support of a community of learners like you. 
 </div>
```
Comment (rosemm) [inline new_to_sql.md:49]:
```suggestion

</div>
```

File: new_to_version_control.md [hunk 8]
```diff
@@ -24,17 +24,14 @@ If you prefer something a bit more "just in time", however, we suggest some reso
...
 * Our interest form at https://redcap.link/dart-interest
 * Taking the screening for participation in the study at https://redcap.link/DART-survey
 * Emailing us at dart@chop.edu
 
-and consider **not** consuming these modules.
-
-Consuming these modules will **render you ineligible to participate** in our research.  However, we know that they're useful on their own, outside of our conduct of research, so we're fast-tracking access to these materials.
-
+If you meet our research criteria and decide to enroll as a study participant, you'll receive additional data science learning support, including a customized curriculum of materials and the peership and support of a community of learners like you. 
 </div>
```
Comment (rosemm) [inline new_to_version_control.md:35]:
```suggestion

</div>
```

**Comment by rosemm** [comment]
Ah yes, I see it now too. I think I just fixed it --- try again?

**Comment by franzenr** [comment]
works for me now!! 🙌 

Although I just realized that in the new box, the links aren't descriptive links, it's just written out, which is not ideal for anyone using a screenreader. So I'm going to hop in and make those changes now since we're already making edits!

**Comment by franzenr** [comment]
Got through all of the "new to" documents, but there's also lots of descriptive link text and alt text issues in the orientation doc -- Don't have the full bandwidth to tackle it right now, so I'm going to file it away as an issue for now. @rosemm

**Pull request merged**